contrib/IECoreUSD/include/IECoreUSD/SceneCacheDataAlgo.h

```diff
@@ -53,12 +53,16 @@ namespace IECoreUSD
 namespace SceneCacheDataAlgo
 {
 
-IECOREUSD_API pxr::TfToken internalRootNameToken();
-IECOREUSD_API IECore::InternedString internalRootName();
+IECOREUSD_API const pxr::TfToken internalRootNameToken();
+IECOREUSD_API const IECore::InternedString internalRootName();
+IECOREUSD_API const pxr::TfToken cortexPrimitiveVariableMetadataToken();
```

Comment:
Yes that's totally out of paranoia and because it returns a static...
Good to know it's not necessary, thanks!

Reply:
Cool - if it was returning a reference to a static, then it would absolutely be necessary to make it const, but since it's returning a value, the caller is getting a copy anyway.

Reply:
addressed here 11e04684d8f8bd176793a38491f2e3f689fa5673